ci/install-dev-deps.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
+# Install developer build dependencies
+
+# This installs tools used by local development only.
+
+set -xeuo pipefail
+
+# cxx.rs (cxxbridge) isn't packaged in Fedora today.  It generates
+# source code, which we vendor along with our dependent crates into release
+# tarballs.
+CXX_VER=$(cargo metadata --format-version 1 | jq -r '.packages[]|select(.name == "cxx").version')
+mkdir -p target
+time cargo install --root=target/cxxbridge cxxbridge-cmd --version "${CXX_VER}"
```

Comment:
Don't we need to remove it from `installdeps.sh`?

Reply:
Yes, done!